src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    value: 'Under construction',
+  },
+  reducers: {
+    check: (state) => state.value,
+  },
```

Comment:
The reducer here should check the status and always return `Under construction`. Please consider doing something similar to this: 
 
         reducers: {
            checkStatus: (state, action) => {
              state.categories =
                action.payload === "Under construction"
                  ? "Under construction"
                  : state.categories
            },
          },

Reply:
Thank you so much for your review, I have made the required adjustments.